corehq/apps/userreports/pillow.py

```diff
@@ -188,6 +188,16 @@ def bootstrap(self, configs=None):
         self.bootstrapped = True
         self.last_bootstrapped = datetime.utcnow()
 
+    @property
+    def relevant_domains(self):
```

Comment:
nit: should this be named `adapters_by_domain`?
`self.table_manager.relevant_domains` makes it look like its going to return domains.

Reply:
it returns a set of domain names

Reply:
Okay, the method is returning `table_adapters_by_domain`, which is possible a dict and you just intend to get the domain which are keys. That makes it clear but is a bit confusing.